packages/marshal/src/marshal.js

```diff
@@ -542,28 +543,45 @@ export function passStyleOf(val) {
  *      same order in which they appeared in the parsed JSON string.
  */
 function makeReplacerIbidTable() {
+  /** @type {Map<object, number>} */
   const ibidMap = new Map();
   let ibidCount = 0;
 
   return harden({
+    /**
+     * @param {object} obj
```

Comment:
Since we are explicitly typing these, we may as well do the return value too.

```suggestion
     * @param {object} obj
     * @returns {boolean}
```
and likewise below.

Reply:
In practice, only parameters need explicit typings, and I prefer minimalism.  If you have a compelling reason to type the return values as well, we can do that, but I think it's mostly clutter.  Users of the implicit returning functions will see the actual type when they hover over the call site.

Reply:
I didn't know that about the hover. @dckc IIRC I gave you a hard time over the same omission. I see now I was mistaken.